rfc9114.md

```diff
@@ -94,12 +95,12 @@ HTTP semantics ({{!SEMANTICS=I-D.ietf-httpbis-semantics}}) are used for a broad
 range of services on the Internet. These semantics have most commonly been used
 with HTTP/1.1 and HTTP/2.  HTTP/1.1 has been used over a variety of transport
 and session layers, while HTTP/2 has been used primarily with TLS over TCP.
-HTTP/3 supports the same semantics over a new transport protocol, QUIC.
+HTTP/3 supports the same semantics over a new transport protocol: QUIC.
 
-## Prior versions of HTTP
+## Prior Versions of HTTP
 
 HTTP/1.1 ({{?HTTP11=I-D.ietf-httpbis-messaging}}) uses whitespace-delimited text
-fields to convey HTTP messages.  While these exchanges are human-readable, using
+fields to convey HTTP messages.  While these exchanges are human readable, using
```

Comment:
FYI https://github.com/NTAP/isb-ietf-config/issues/12

Reply:
@MikeBishop want to close NTAP/isb-ietf-config#11 now?